security/calico-enterprise/egress-access-control.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Advanced egress access controls with Calico Enterprise
+description: Calico Enterprise provides fine-grained access control between a pod and external services outside using DNS.
+calico_enterprise: true
+---
+
+### Egress access control to external endpoints
+
+Fine-grained access control between a Pod and external service outside the cluster generally requires a firewall rule or equivalent between your Pod and the services you need to connect to. Since traditional firewall rules are defined using static IP addresses, the dynamic nature of Pod IP assignment creates challenges defining firewall rules.
```

Comment:
The capitalization of `Pod` is a bit unsual. I don't think we do that elsewhere in our docs.

Reply:
I agree. I was loathe to change anything that Andy and Alex approved in pure Marketing material. Leave as is.